python/ray/services.py

```diff
@@ -553,13 +558,30 @@ def start_redis(node_ip_address,
     return redis_address, redis_shards
 
 
+def _make_temp_redis_config(node_ip_address):
+    """Create a configuration file for Redis.
+
+    Args:
+        node_ip_address: The IP address of this node. This should not be
+            127.0.0.1.
+    """
+    redis_config_name = "/tmp/redis_conf{}".format(random_name())
```

Comment:
Nobody cleans up this file, no? :-)

This is really what [`NamedTemporaryFile`](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) was designed to do.